<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lecturer_Student_Record</class>
 <widget class="QMainWindow" name="Lecturer_Student_Record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="headerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="headerHud">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/header_short.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/back.png</normaloff>:/res/assets/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="headerSubjectCodeLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>15</y>
       <width>310</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <fontweight>DemiBold</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFF;
	text-align: center;
	font-family: Poppins;
	font-size: 24px;
	font-style: normal;
	font-weight: 600;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>SC&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headerSubjectDescLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>45</y>
       <width>310</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 9px;
	font-style: normal;
	font-weight: 600;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>SD&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headerProgramLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>65</y>
       <width>310</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 400;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>P&lt;?&gt; YS&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="footerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>360</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="footerHud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>340</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/footer.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/home_unselected.png</normaloff>:/res/assets/home_unselected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="homeLabel">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>105</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="classListButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/class_list_selected.png</normaloff>:/res/assets/class_list_selected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="classListLabel">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>105</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>CLASS LIST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="whiteBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>130</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/white_box.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>whiteBox</zorder>
    <zorder>footerHud</zorder>
    <zorder>homeButton</zorder>
    <zorder>homeLabel</zorder>
    <zorder>classListButton</zorder>
    <zorder>classListLabel</zorder>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>110</y>
      <width>180</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #000F2A;
	font-family: Poppins;
	font-size: 13px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
    </property>
    <property name="text">
     <string>STUDENT RECORD</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="datetimeGroup">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>140</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="calendarIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/calendar.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>5</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #000F2A;
	font-family: Poppins;
	font-size: 9px;
	font-style: normal;
	font-weight: 300;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Date&lt;?&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>15</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #000F2A;
	font-family: Poppins;
	font-size: 11px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Time&lt;?&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="studentsListArea">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>190</y>
      <width>310</width>
      <height>330</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="contents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>308</width>
       <height>328</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/res/assets/download.png</normaloff>:/res/assets/download.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>headerGroup</zorder>
   <zorder>titleLabel</zorder>
   <zorder>datetimeGroup</zorder>
   <zorder>studentsListArea</zorder>
   <zorder>footerGroup</zorder>
   <zorder>downloadButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
